### DISCUSSION TITLE
Failed to get PostgreSQL support info

### DISCUSSION THREAD

**Discussion by antmeehan** [ravendb/ravendb#15908] [Q&A]
After upgrading from v4.2 to v5.2 server, we can no longer access the ETL in Studio. When clicking on "Integrations" within a database, we get the following error:

> You have enabled the PostgreSQL integration via 'Integrations.PostgreSQL.Enabled' configuration but this is an experimental feature and the server does not support experimental features. Please enable experimental features by changing 'Features.Availability' configuration value to 'Experimental'.

I've tried adding the `Integrations.PostgreSQL.Enabled` setting as `false` in `settings.json`, but that hasn't helped, and doesn't seem to be picked up when looking at the Server Settings UI.

We are using a Professional license.

Any ideas?

**Comment by arekpalinski** [comment]
Hi,

1. PostgreSQL integration feature (https://ravendb.net/docs/article-page/5.4/csharp/integrations/postgresql-protocol/overview) wasn't available in v4.2. You mentioned ETL so maybe you mean SQL ETL feature (https://ravendb.net/docs/article-page/5.4/csharp/server/ongoing-tasks/etl/sql)?

2. If you really want to use PostgreSQL integration you should set the following in settings.json file
```
"Integrations.PostgreSQL.Enabled": true,
"Features.Availability": "Experimental"
```

**Reply by antmeehan** [reply]
Sorry yes we've only got SQL ETL defined. But now when accessing the Integrations page it fails with this error and we can't access it.

**Reply by arekpalinski** [reply]
SQL ETL isn't part of Integrations page. You'll find it in Ongoing Tasks:

![image](https://user-images.githubusercontent.com/376551/221779022-eeb7f7b3-414d-4afa-b367-cfcedd94d1cb.png)

**Reply by antmeehan** [reply]
Oh my bad! So sorry. I can see it now. 😳
